lib/ddtrace/contrib/rake/integration.rb

```diff
@@ -19,10 +19,6 @@ def self.present?
           super && defined?(::Rake)
         end
 
-        def self.compatible?
-          super && RUBY_VERSION >= '2.0.0'
-        end
```

Comment:
This method is replicating the behaviour of `super`, which is responsible for checking the minimum Ruby version.